addons/mass_mailing_crm/views/mailing_mailing_views.xml

```diff
@@ -22,4 +22,15 @@
     		</xpath>
         </field>
     </record>
+
+    <record id="crm_lead_view_search_from_mailing" model="ir.ui.view">
+        <field name="name">crm.lead.view.search.from.mailing</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.view_crm_case_opportunities_filter"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group/filter[@name='month']" position="replace">
+                <filter string="Creation Date" context="{'group_by':'create_date:day'}" name="day"/>
```

Comment:
Still wondering why we change this filter globally with mass mailing. Either this is primary (specific to mass mailing), either this should be done in base CRM module. Strange that having the bridge of mass mailing and crm changes the way "creation date" filter works.

Reply:
This should absolutely be primary, it is indeed specific to Mass Mailing, and shouldn't affect CRM itself.